bindings/node/index.js

```
@@ -1,6 +1,10 @@
 const root = require("path").join(__dirname, "..", "..");
 
-module.exports = require("node-gyp-build")(root);
+module.exports =
+  typeof process.versions.bun === "string"
```

ESLint

[error] Strings must use singlequote.

(quotes)